frontend/src/app/pages/exercises/exercise/shared/exercise-map/feature-managers/material-feature-manager.ts

```diff
@@ -44,6 +51,21 @@ export class MaterialFeatureManager extends MoveableFeatureManager<Material> {
 
     private readonly popupHelper = new ImagePopupHelper(this.olMap, this.layer);
 
+    private readonly openPopupCircleStyleHelper = new CircleStyleHelper(
+        (feature) => ({
```

Comment:
This parameter is not used. Please omit it, replace it by `_` or rename it to `_feature`.

Reply:
Done